cmd/bbolt/main.go

```diff
@@ -921,6 +924,11 @@ func (cmd *keysCommand) Run(args ...string) error {
 
 	// Require database path and bucket.
 	relevantArgs := fs.Args()
+	if len(relevantArgs) < 2 {
+		err := fmt.Errorf("Error: %w.\n", ErrNotEnoughArgs)
+		fmt.Fprintln(cmd.Stderr, cmd.Usage())
+		return err
```

Comment:
```suggestion
		return ErrNotEnoughArgs
```

Reply:
updated, but this way the `usage` will not be printed

Reply:
Your previous implement (see below) isn't consistent with the cobra style command's output. 

Overall it isn't a big deal for now. Let's refactor it in future.

```
$ ./bbolt keys
usage: bolt keys PATH [BUCKET...]

Print a list of keys in the given (sub)bucket.
=======

Additional options include:

	--format
		Output format. One of: auto|ascii-encoded|hex|bytes|redacted (default=auto)

Print a list of keys in the given bucket.

Error: not enough arguments.
```